tensorboard/test_util.py

```diff
@@ -22,8 +22,31 @@
 from __future__ import division
 from __future__ import print_function
 
+import logging
 import threading
 
+import tensorflow as tf
+
+from tensorboard import util
+
+
+class TestCase(tf.test.TestCase):
+  """TensorBoard base test class.
+
+  This class enables logging and prints test method names.
```

Comment:
If a test method fails, then Bazel prints the logs for all test methods, even the ones that passed, and doesn't separate them, so it's not readable. Internally, we already do sort of what this class does.